tests/supports/AssertionException.php

```diff
@@ -6,9 +6,9 @@
 
 class AssertionException extends Exception
 {
-    public function __construct($expectedValue, $actualValue)
+    public function __construct($expectedValue, $actualValue, $message = null)
     {
-        $message = 'Assertion error:' . PHP_EOL . 'Expected: ' . PHP_EOL . json_encode($expectedValue, JSON_PRETTY_PRINT) . PHP_EOL . ' Actual: ' . PHP_EOL . json_encode($actualValue, JSON_PRETTY_PRINT) . PHP_EOL;
+        $message = (is_null($message)) ? 'Assertion error:' . PHP_EOL . 'Expected: ' . PHP_EOL . json_encode($expectedValue, JSON_PRETTY_PRINT) . PHP_EOL . ' Actual: ' . PHP_EOL . json_encode($actualValue, JSON_PRETTY_PRINT) . PHP_EOL : $message;
```

Comment:
let's move `PHP_EOL . json_encode($expectedValue, JSON_PRETTY_PRINT) . PHP_EOL` and second argument to the separate variables.